document.txt

```diff
@@ -0,0 +1,8 @@
+This is an important document 
+-------------
+
+Here's an important list:

```

Comment:
Testing reply.

---
In reply to: [59109388](https://github.com/kivancmuslu/CFTest/pull/2#discussion_r59109388) [](ancestors = 59109388)